index.html

```diff
@@ -58,59 +58,59 @@ <h3 class="h3">
       </section>
 
       <section id="body2" class="body2">
-
+        
         <div id="sec2">
         <h2 class="s2h1">My Recent Works</h2>
         <hr class="hr1" />
       </div>
-
+      
+       <!-- <div class="mainbody"> -->
```

Comment:
- [ ] Pop-up window is looking great! However, as the previous reviewer said, the content of the pop-up window and the works section should be created dynamically, this means that all the hardcoded info from line 66 to 253 should be deleted and it should be loaded using DOM manipulation with JavaScript. The goal is to have an array with all the data from the projects and create a function that adds all the HTML elements to the DOM. 

Please remove all the hardcoded info from the HTML and use JS to load it dynamically.